LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2016 Brightcove, Inc.
+Copyright Gary Katsevman <me@gkatsev.com>
```

Comment:
Ah, yes. It's because you're the `author` on this repo. I'll change that instead.